bioimageio_collection_config.json

```diff
@@ -506,6 +506,34 @@
                     "tags": [
                         "SpotMAX"
                     ]
+                },
+                {
+                    "background_image": "static/img/zoo-background.svg",
+                    "contact": [
+                        {
+                            "affiliation": [
+                                {
+                                    "name": "Human Technopole",
+                                    "url": "https://humantechnopole.it/en/"
+                                }
+                            ],
+                            "email": "rse@fht.org",
+                            "github": "https://github.com/orgs/CAREamics/teams/maintainers",
+                            "name": "CAREamics Maintainers"
+                        }
+                    ],
+                    "default_type": "model",
+                    "docs": "https://careamics.github.io/",
+                    "explore_button_text": "Start Exploring",
+                    "icon": "https://github.com/CAREamics/.github/blob/main/profile/images/logo_careamics_128.png",
```

Comment:
Shouldn't the link be to the raw file https://raw.githubusercontent.com/CAREamics/.github/refs/heads/main/profile/images/logo_careamics_128.png?

Reply:
oops, thanks, will update